src/cli/cbenv_register.rs

```diff
@@ -42,6 +42,18 @@ impl Command for CbEnvRegister {
                 SyntaxShape::String,
                 "the connection string to use for this cluster",
             )
+            .named(
+                "username",
+                SyntaxShape::String,
+                "the username to use for the registered cluster",
+                None,
```

Comment:
We haven't really in the past, but there is no reason I'm aware of. I'll add some short options for password and username since shortening them is pretty standard.

Reply:
Did you mean to remove the shortened fields again?

Reply:
Yeah I made them `-u` and `-p` nut then thought that we'd want to reserve p for the shortening of the project flag since this will be more common

Reply:
Makes sense